logsearchapi/docker-compose.yaml

```diff
@@ -0,0 +1,58 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:13-alpine
+    container_name: postgres-db
+    restart: always
+    command: -c "log_statement=all"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - '5432:5432'
+    volumes:
+      - db:/var/lib/postgresql/data
+
+  api:
+    image: ahmedzidan204/logsearchapi:v1.0.0 #change to 0chain registry
```

Comment:
Use docker image from 0chaindev

Reply:
@ahmedzidan

Reply:
This file will be deleted, I've already pushed the latest docker-compose file in environment folder

Reply:
Hi @peterlimg thanks for your comments, let me reply  to them here
1- Use cmd for logsearchapi and client API?  these are two different services has nothing to do with the core package of minioserver, please have a look here to minioOperator repo and see where they put the logsearchapi. https://github.com/minio/operator/tree/master/logsearchapi

Reply:
2-  use snake case? yeap sure we can use that I can change it